src/components/Calculator.css

```diff
@@ -48,5 +48,5 @@ input[type='text']::placeholder {
 }
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍